salt/loader.py

```diff
@@ -444,6 +444,15 @@ def auth(opts, whitelist=None):
     )
 
 
+def schemas(opts, whitelist=None):
```

Comment:
I have mixed feelings on this, I like that it is consistent with all other loaders but given that this is now in the hot path of every event being emitted, it may cause performance issues.

Reply:
Using a loader here is the right approach, this does not cause me concern